app/lib/realm.js

```diff
@@ -468,7 +469,7 @@ class DB {
 	setActiveDB(database = '') {
 		const path = database.replace(/(^\w+:|^)\/\//, '');
 		return this.databases.activeDB = new Realm({
-			path: `${ path }.realm`,
+			path: `${ RNRealmPath.realmPath }${ path }.realm`,
```

Comment:
Is this going to find the correct path or it's going to logout all of our users?

Reply:
to access same database between client and share-extension on iOS we need to put that to save realmDB on the directory of app group, if this is android returns ```""```.

Reply:
This not going to logout Native users.